app/src/main/java/com/example/encoratask/service/CharacterService.kt

```diff
@@ -0,0 +1,27 @@
+package com.example.encoratask.service
+
+import android.content.Context
+import com.android.volley.Request
+import com.android.volley.Response
+import com.android.volley.toolbox.StringRequest
+import com.android.volley.toolbox.Volley
+
+/**
+ * Created by Erick Sanchez
+ * Revision 1 - 10/06/21
+ */
+open class CharacterService{
+
+    companion object {
+
+        private val apiUrl = "https://rickandmortyapi.com/api"
+
+        fun getCharacters(context:Context, response:Response.Listener<String>, error: Response.ErrorListener) {
+
+            val url = "$apiUrl/character/"
+            val mRequestQueue = Volley.newRequestQueue(context)
```

Comment:
hungarian notation is not used anymore

Reply:
Ok, sorry, i use to use in Java, and it works pretty fine in the community, sorry, I search in Google and fopro the Kotlin comunnity its a horrible practive, really sorry